socorro/unittest/external/es/base.py

```diff
@@ -301,11 +311,10 @@
         'permissions_needed': [],
         'query_type': 'string',
         'storage_mapping': {
-            'type': 'string',
+            'type': 'text',
             'fields': {
                 'full': {
-                    'type': 'string',
-                    'index': 'not_analyzed',
+                    'type': 'keyword',
```

Comment:
However you were right that some of those choices I made where dumb, so I went through all of them and fixed a few. Now all of them should be coherent with the data we have in production.